ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/polygon/PolygonDrawingTaskFragment.kt

```diff
@@ -48,13 +50,10 @@ class PolygonDrawingTaskFragment : Hilt_PolygonDrawingTaskFragment<PolygonDrawin
 
   override fun onCreateTaskBody(inflater: LayoutInflater): View {
     val rowLayout = LinearLayout(requireContext()).apply { id = View.generateViewId() }
+    polygonDrawingMapFragment = PolygonDrawingMapFragment.newInstance(viewModel, map)
     parentFragmentManager
       .beginTransaction()
-      .add(
-        rowLayout.id,
-        PolygonDrawingMapFragment.newInstance(viewModel, map),
-        "Draw a polygon fragment"
-      )
+      .add(rowLayout.id, polygonDrawingMapFragment, "Draw a polygon fragment")
```

Comment:
Same here - better to use class name or to omit.

Reply:
Done